client/src/app/infrastructure/utils/functions.ts

```diff
@@ -312,8 +312,11 @@ export function objectToFormattedString(jsonOrObject: string | object): string {
     if (!jsonOrObject) {
         return undefined;
     }
-    let json =
-        jsonOrObject && typeof jsonOrObject !== `string` ? JSON.stringify(jsonOrObject) : (jsonOrObject as string);
+    let json = (
+        jsonOrObject && typeof jsonOrObject !== `string` ? JSON.stringify(jsonOrObject) : (jsonOrObject as string)
+    )
+        .split(/\n|\s/)
+        .join(``);
```

Comment:
I don't think this is a good idea as this could also change content of properties.

Reply:
This function is supposed to format object json into human-readable strings, taking out pre-existing strings and line-breaks before adding in the desirable ones is pretty necessary IMO.

I don't think the result of this is going to be saved anywhere where it wouldn't be fine to save the transformed value.

Or did you mean something else?